addons/common/fnc_getItemConfig.sqf

```diff
@@ -23,16 +23,22 @@ Author:
 ---------------------------------------------------------------------------- */
 SCRIPT(getItemConfig);
 
-params [["_weapon", "", [""]]];
+params [["_item", "", [""]]];
 
 private _result = configNull;
 
 {
-    private _config = configFile >> _x >> _weapon;
+    private _config = configFile >> _x >> _item;
 
     if (isClass _config) exitWith {
         _result = _config;
     };
 } forEach ["CfgWeapons", "CfgMagazines", "CfgGlasses"];
 
+private _config = configFile >> "CfgVehicles" >> _item;
```

Comment:
Only do this entire new section if result is null still?

Reply:
I suppose?

It wouldn't change the return value though.

Reply:
It would be mildly faster for non-backpack cases.

Reply:
Yes, and slower for backpack classes.

Reply:
One would think that's least used.